extensions/amp-animation/amp-animation.md

```diff
@@ -299,6 +333,64 @@ can be reduced to an array of components. For instance:
 </amp-animation>
 ```
 
+
+### Animation composition
```

Comment:
Added possible reasons. Similar to function call composition, two reasons: allow reuse across multiple animations (as with calling function from different places) or simply making each animation more manageable (as with splitting unwieldy function into two, even if there's no reuse).